packages/grid/_modules_/grid/models/gridOptions.tsx

```diff
@@ -87,6 +88,18 @@ export enum DensityTypes {
   Comfortable = 'comfortable',
 }
 
+/**
+ * Sets the types of the texts in the grid.
+ */
+export interface LocaleText {
+  // Density selector toolbar button text
```

Comment:
That's the interface I mean

Reply:
It should be a model

Reply:
Changing it to a type will still result in the problem mentioned above, because entries will still be optional.

Reply:
Part of the previous thread - updated